packages/amtk.rb

```diff
@@ -0,0 +1,28 @@
+require 'package'
+
+class Amtk < Package
+  description 'Actions, Menus and Toolbars Kit for GTK+ applications'
+  homepage 'https://wiki.gnome.org/Projects/Amtk'
+  version '5.2.1'
```

Comment:
Shouldn't this version be the first few characters of the git hash instead?

Reply:
Yeah that would make sense.